### DISCUSSION TITLE
How to used fit_pseudolabel

### DISCUSSION THREAD

**Discussion by talhaanwarch** [autogluon/autogluon#2004] [General]
I am but confuse on how to use fit_pseudolabel.
I tried this way
predictor=TabularPredictor(...).fit(....)
then predictor.fit_pseudolabel(...)
but the result before and after fit_pseudolabel are identical.
May be when i called predictor.predict() its referring to predictor before fit_pseudolabel

**Comment by Innixma** [comment]
You may want to call `predictor.leaderboard(test_data)` to see if new models were trained.

This code was written by an intern and we haven't yet gotten around to writing documentation / fleshing out support for it yet, so consider it experimental for the time being.
